projects/packages/waf/src/class-brute-force-protection.php

```diff
@@ -547,11 +547,16 @@ public function log_failed_attempt( $username, $error = null ) {
 		if ( isset( $_COOKIE['jpp_math_pass'] ) ) {
 
 			$transient = $this->get_transient( 'jpp_math_pass_' . sanitize_key( $_COOKIE['jpp_math_pass'] ) );
-			--$transient;
+			if ( is_int( $transient ) ) {
+				--$transient;
+			}
 
-			if ( ! $transient || $transient < 1 ) {
+			if ( ! $transient || ! is_int( $transient ) || $transient < 1 ) {
```

Comment:
A nitpick and non-blocker: I believe the first condition is now redundant:

```suggestion
			if ( ! is_int( $transient ) || $transient < 1 ) {
```